File: alzir/Entregável #1 - Algoritmos/Java/src/Fibonacci.java
import java.util.Scanner;

public class Fibonacci {

  public static int fibonacci(int num) {
    if (num < 2) {
      return num;
    } else {
      return fibonacci(num - 1) + fibonacci(num - 2);
    }
  }

  public static void main(String[] args) {
    Scanner sc = new Scanner(System.in);
    System.out.print("Digite um número: ");
    int n = sc.nextInt();
    if (n <= 0) {
      System.out.println("Digite um número inteiro positivo!");
    } else {
      for (int i = 0; i < n; i++) {
        System.out.println(fibonacci(i));
      }
    }
    sc.close();
  }
}

File: alzir/Entregável #1 - Algoritmos/JavaScript/fibonacci.js
const readline = require("readline").createInterface({
  input: process.stdin,
  output: process.stdout,
});

const fibonacci = (num) => {
  if (num < 2) {
    return num;
  } else {
    return fibonacci(num - 1) + fibonacci(num - 2);
  }
};

readline.question("Digite um número: ", (n) => {
  if (n <= 0 || isNaN(n)) {
    console.log("Digite um número inteiro positivo!");
  } else {
    for (let i = 0; i < n; i++) {
      readline.close();
      console.log(fibonacci(i));
    }
  }
});
